v0/destinations/posthog/transform.js

```diff
@@ -79,6 +79,19 @@ const responseBuilderSimple = (message, category, destination) => {
     ...payload.properties
   };
 
+  // This is to ensure groupType, groupKey delete from $group_set, as it is properly mapped
+  // in 'properties.$group_type' and 'properties.$group_key' respectively.
+  if (
+    payload.properties &&
+    isDefinedAndNotNull(payload.properties.$group_set)
+  ) {
+    if (isDefinedAndNotNull(payload.properties.$group_set.groupType))
+      delete payload.properties.$group_set.groupType;
+
+    if (isDefinedAndNotNull(payload.properties.$group_set.groupKey))
+      delete payload.properties.$group_set.groupKey;
```

Comment:
Remove this, as we are mapping groupId to $group_key.

Reply:
Yeah, updated.